data/seeds/000-cleanup.js

```diff
@@ -0,0 +1,5 @@
+const knexCleaner = require('knex-cleaner');
+
+exports.seed = function(knex) {
+	return knexCleaner.clean(knex);
```

Comment:
I'm not so sure that you need this since you're using `truncate` for your other seeds.